src/MaxText/layers/moe.py

```diff
@@ -809,50 +808,41 @@ def gmm(inputs, kernel, tiling, group_sizes, expert_assignments):
           min(tiling[1], k),
           min(tiling[2], n),
       )
-      if self.config.use_tokamax_gmm:
-        output = tokamax.ragged_dot(
+      if self.config.megablox:
```

Comment:
Oh, please be careful about this change. We previously update to `use_tokamax_gmm` for less confusion in configs.